gapic-generator-java-pom-parent/pom.xml

```diff
@@ -95,8 +95,86 @@
         </plugins>
       </build>
     </profile>
+
+    <profile>
+      <id>test-coverage</id>
```

Comment:
The PR description does a great job clarifying how this is meant to be used, but that may get lost in obscurity once the PR is merged. Could you add some information in an appropriate `.md` to explain the different expected workflows + report options?

Reply:
Good idea, done. I've added a README to the `coverage-report` module.